Core/Core/View/Base/HTMLFormattedText.swift

```diff
@@ -37,6 +37,7 @@ public struct HTMLFormattedText: UIViewRepresentable {
         textView.textContainerInset = .zero
         textView.textContainer.lineFragmentPadding = 0
         textView.delegate = context.coordinator
+        textView.accessibilityIdentifier = "honer_code_textarea"
```

Comment:
Suggestion to replace identifier with `honer_code_text_area`

Reply:
`textarea` is used for field type like textfield.